tools/group_humann2_uniref_abundances_to_go/.shed.yml

```diff
@@ -0,0 +1,14 @@
+categories: 
+  - Metagenomics
+description: "Group abundances of UniRef50 gene families obtained with HUMAnN2 to Gene Ontology (GO) slim terms with relative abundances"
+name: group_humann2_uniref_abundances_to_go
+owner: bebatut
```

Comment:
This looks wrong here ...

Oh wait, now I see this tool exists already ... why not updating the one in Bérénice repository?

If we add one here as GalaxyP, this will then be two __different__ tools in Galaxy as the owner is different.

Reply:
Ok. I will follow up on my PR for Bérénice.

Reply:
I ask her as well, she will be back in 2 days, can this PR here wait for 2 days. ping @bebatut

Reply:
Sounds good. We can do that.